tools/nmr_bucketing/NmrBucketing_xml.xml

```diff
@@ -0,0 +1,302 @@
+                <option value="yes" > yes </option>
+                <option value="no" selected="true"> no </option>
+            </param>
+            <when value="yes">
+                <repeat name="conditions" title="exclusion zones">
+                    <param name="zone_exclusion_left" label="Left exclusion zone border" type="float" value="10.0" />
+                    <param name="zone_exclusion_right" label="Right exclusion zone border" type="float" value="10.0" />
+                </repeat>
+            </when>
+            <when value="no">
+            </when>
+        </conditional>
+
+        <param name="graphType" label="Spectra representation" type="select" help="Select 'None' for no representation,'Overlay' to overlay all spectra on a unique chart and 'One per individual' to generate an individual chart for each observation">
+            <option value="None"> none </option>
+            <option value="Overlay"> Overlay </option>
+            <option value="One_per_individual"> One_per_individual </option>
+        </param>
+
+    </inputs>
+
+    <outputs>
+		<data format="txt" name="logOut" label="${tool.name}_log" />
+        <data format="tabular" name="sampleOut" label="${tool.name}_sampleMetadata" />
+        <data format="tabular" name="variableOut" label="${tool.name}_variableMetadata" />
+        <data format="tabular" name="dataMatrixOut" label="${tool.name}_bucketedData" />
+        <data format="pdf" name="graphOut" label="${tool.name}_spectra" >
+            <filter> graphType != "None" </filter>
+        </data>
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="inputs|input" value="zip_file" />
+            <param name="inputs|zip_file" value="MTBLS1.zip"  ftype="zip" />
+            <param name="bucket_width" value="0.01" />
+            <param name="left_border" value="9.3" />
+            <param name="right_border" value="0.8" />
+            <param name="zone_exclusion_choices|choice" value="yes" />
+            <param name="zone_exclusion_choices|conditions_0|zone_exclusion_left" value="6.0" />
+            <param name="zone_exclusion_choices|conditions_0|zone_exclusion_right" value="4.24" />
+            <param name="zone_exclusion_choices|conditions_1|zone_exclusion_left" value="3.99" />
+            <param name="zone_exclusion_choices|conditions_1|zone_exclusion_right" value="3.19" />
+            <param name="graphType" value="Overlay" />
+            <output name="dataMatrixOut" file="MTBLS1_bucketedData.tabular" />
+            <output name="sampleOut" file="MTBLS1_sampleMetadata.tabular" />
+            <output name="variableOut" file="MTBLS1_variableMetadata.tabular" />
+        </test>
+    </tests>
+
+    <help>
+
+.. class:: infomark
+
+**Authors** Marie Tremblay-Franco (marie.tremblay-franco@toulouse.inra.fr), Marion Landi (marion.landi@clermont.inra.fr) and Franck Giacomoni (fgiacomoni@clermont.inra.fr)

```

Comment:
Please use the more explicit schema.org annotations for this: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator-person

The advantage is that Galaxy can nicely render this content and you can access it via the API and such ....